metadata-io/src/main/java/com/linkedin/metadata/graph/neo4j/Neo4jGraphService.java

```diff
@@ -94,9 +95,9 @@ public void addEdge(@Nonnull final Edge edge) {
 
   @Nonnull
   public RelatedEntitiesResult findRelatedEntities(
-      @Nullable final String sourceType,
+      @Nullable final List<String> sourceTypes,
```

Comment:
should we keep nullable? if we can avoid less program states we should go for it!

Reply:
yes, null is going to represent `all` as it has historically